README.md

```diff
@@ -10,46 +10,93 @@ This project uses [uv](https://github.com/astral-sh/uv).
 
 Thunderbird Accounts is a Django 5.1 application. Please read up on Django before diving in.
 
-## Setup / Running
+Docker is the recommended way to run the server, though you have the option to run it locally.
 
-### Installing
 
-Ensure that uv is installed, and then run:
+## Shared setup
+
+Whether you're running the server via Docker or locally, you'll need to set up your `.env` file and copy the example Stalwart config.
+
+### Create the .env file
+
+Start from the `.env.example` file to create your dev and test `.env` files:
+
 ```shell
-uv sync
+cp .env.example .env
+cp .env.example .env.test
 ```
 
-### Using the example config for Stalwart
+In `.env.test`, set `APP_ENV=test`
```

Comment:
.env.test is safe to commit to the repo as long as we don't have any live keys in it. So I can get that up later today.